### PR TITLE
Ask a Frontend Developer for help

### PR DESCRIPTION
**Pull request by domoscargin**
Adds details for getting help from a Frontend Developer.

### CODE REVIEW

**Comment by kellylee-gds** [comment]
Thanks for drafting this @domoscargin 🎉

**Comment by domoscargin** [comment]
Thinking I might split out the more general changes from the get help from a developer stuff

File: source/how-we-work/get-help-from-other-roles.html.md [hunk 1]
```diff
@@ -38,7 +39,31 @@ You can ask a product manager and delivery manager for help with
...
 - send a direct message on Slack
 - use a collaborative tool such as Padlet, Trello, Mural or Google Doc to frame your thoughts
-- arrange a call  
+- arrange a call
+
+## Frontend developer
+
+A frontend developer designs, builds and improves website software that meets user needs. They write clean, accessible and open code, and are knowledgeable about fundamental frontend technologies such as HTML, CSS and JavaScript. You can read more about [frontend developers’](https://ddat-capability-framework.service.gov.uk/frontend-developer.html) skills on the DDaT Capability Framework.
+
+On the Design System team, frontend developers build and maintain [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/), the [Design System Website](https://github.com/alphagov/govuk-design-system) and the [rest of our product repositories](/what-we-do/#our-products).
```
Comment (colinrotherham) [inline source/how-we-work/get-help-from-other-roles.html.md:51]: Think we usually say ~Website~ website, but super minor

```suggestion
On the Design System team, frontend developers build and maintain [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/), the [Design System website](https://github.com/alphagov/govuk-design-system) and the [rest of our product repositories](/what-we-do/#our-products).
```

File: source/how-we-work/get-help-from-other-roles.html.md [hunk 2]
```diff
@@ -38,7 +39,31 @@ You can ask a product manager and delivery manager for help with
...
+
+- using GitHub
+- investigating and implementing automation on GitHub
+- transforming designs into working HTML pages
+- the feasibility of designs when translated to code
+- HTML, CSS and JavaScript
+- accessibility best practices
+- web performance
+- Setting up our projects and their tooling on your machine
+- Troubleshooting errors when running our projects (both on your machine and GitHub)
```
Comment (colinrotherham) [inline source/how-we-work/get-help-from-other-roles.html.md:63]: Do these two need to match the others?

```suggestion
- setting up our projects and their tooling on your machine
- troubleshooting errors when running our projects (both on your machine and GitHub)
```

Comment (colinrotherham) [reply source/how-we-work/get-help-from-other-roles.html.md:63]: Do we say computer instead of machine?

**Pull request merged**